pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -58,9 +58,17 @@
         [10, np.nan, 20],
         np.dtype("float"),
         "Int64",
-        {("convert_integer", False): np.dtype("float")},
+        {
+            ("convert_integer", False, "convert_floating", True): "Float64",
+            ("convert_integer", False, "convert_floating", False): np.dtype("float"),
```

Comment:
@jreback this is the test case with floats that are int-like. By default it gets casted to Int64, but if `convert_integer=False` to Float64